packages/odyssey-icons/figma/close.svg

```diff
@@ -1,3 +1,3 @@
-<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M19.701 5.83047C20.0997 5.43176 20.0997 4.78532 19.701 4.38661L19.6134 4.29903C19.2147 3.90032 18.5682 3.90032 18.1695 4.29903L12 10.4686L5.83047 4.29903C5.43176 3.90032 4.78533 3.90032 4.38662 4.29903L4.29903 4.38662C3.90033 4.78532 3.90033 5.43176 4.29903 5.83047L10.4686 12L4.29903 18.1695C3.90032 18.5682 3.90032 19.2147 4.29903 19.6134L4.38661 19.701C4.78532 20.0997 5.43176 20.0997 5.83047 19.701L12 13.5314L18.1695 19.701C18.5682 20.0997 19.2147 20.0997 19.6134 19.701L19.701 19.6134C20.0997 19.2147 20.0997 18.5682 19.701 18.1695L13.5314 12L19.701 5.83047Z" fill="#212126"/>
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Maybe I'm wrong, but I always thought the `viewBox` duplicative of the width/height. SVGO should provide an option to remove this IIRC.

Reply:
While we could solve this within our component libraries, I'm curious if we can be flexible with the sizes here.

Is it possible to remove all width/height attributes for the SVG items and _instead_ rely on scoped SCSS? That would allow us to pass sizes through our components instead of fetching/modifying the element via JS.

Reply:
We're tearing out width and height via SVGO, but `viewbox` is a 💯 property. Instead of acting as sizing guidelines, it provides aspect ratio guidance for the vector. That way, whatever we scale the image up/down to, its paths will keep their shape.

CSS Tricks has a solid breakdown: https://css-tricks.com/scale-svg/ (a bit old, but not much has changed in the SVG world)